ecl/hql/hqlexpr.cpp

```diff
@@ -6614,6 +6615,27 @@ IHqlExpression *CHqlRecord::lookupSymbol(IIdAtom * fieldName)
     return ret;
 }
 
+IHqlExpression * CHqlRecord::lookupNearestSymbol(IIdAtom * fieldName)
+{
+    NearestSymbol nearest(fieldName);
+    return LINK(lookupNearestSymbol(fieldName, nearest));
+}
+
+IHqlExpression * CHqlRecord::lookupNearestSymbol(IIdAtom * fieldName, NearestSymbol & nearest)
```

Comment:
Better to not return the symbol - since it is always available from the parameter.  One minor danger is that lookupSymbol() returns a linked expression, while this doesn't - so potential scope for confusion.